app/graphql/filter_helpers.rb

```diff
@@ -1,6 +1,7 @@
 module FilterHelpers
   def where(filters)
-    filters.to_h.with_indifferent_access
+    processed_filters = process_filters(filters)
```

Comment:
FilterHelpers is used in multiple places, so I think we should keep it generic. Let's move your changes into query_type, like so:

```ruby
def providers(filters: nil, order_by: nil)
  updated_within_days = filters.delete(:updated_within_days)
  scope = ::Provider.where(where(filters)).order(order(order_by))
  if updated_within_days
    scope = scope.where("updated_at > ?", updated_within_days.days.ago)
  end
  scope
end
```

(Untested, but I think it's right.)